cannot convert ‘std::nullptr_t’ to ‘VkSurfaceKHR’ on i686

Starting with gamescope 3.12.0 (regressed compared to previously built 3.11.49 on GCC 12), the build (GCC 13, Fedora) on i686 is failing with:

```
../src/main.cpp: In function ‘int main(int, char**)’:
../src/main.cpp:665:32: error: cannot convert ‘std::nullptr_t’ to ‘VkSurfaceKHR’ {aka ‘long long unsigned int’} in initialization
  665 |         VkSurfaceKHR surface = nullptr;
      |                                ^~~~~~~
```

Entire build log can be found here: https://kojipkgs.fedoraproject.org//work/tasks/2329/103912329/build.log

Just dropped it here before calling it a day, can bisect/look at it some more if desired.

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -662,7 +662,7 @@ int main(int argc, char **argv)
 	}
 
 	VkInstance instance = vulkan_create_instance();
-	VkSurfaceKHR surface = nullptr;
+	VkSurfaceKHR surface = VK_NULL_HANDLE;
 
 	if ( !BIsNested() )
 	{

--- a/src/rendervulkan.cpp
+++ b/src/rendervulkan.cpp
@@ -1751,11 +1751,11 @@ void CVulkanCmdBuffer::dispatch(uint32_t x, uint32_t y, uint32_t z)
 		shaperLutDescriptor[i].imageLayout = VK_IMAGE_LAYOUT_SHADER_READ_ONLY_OPTIMAL;
 		// TODO(Josh): I hate the fact that srgbView = view *as* raw srgb and treat as linear.
 		// I need to change this, it's so utterly stupid and confusing.
-		shaperLutDescriptor[i].imageView = m_shaperLut[i] ? m_shaperLut[i]->srgbView() : nullptr;
+		shaperLutDescriptor[i].imageView = m_shaperLut[i] ? m_shaperLut[i]->srgbView() : VK_NULL_HANDLE;
 
 		lut3DDescriptor[i].sampler = m_device->sampler(nearestState);
 		lut3DDescriptor[i].imageLayout = VK_IMAGE_LAYOUT_SHADER_READ_ONLY_OPTIMAL;
-		lut3DDescriptor[i].imageView = m_lut3D[i] ? m_lut3D[i]->srgbView() : nullptr;
+		lut3DDescriptor[i].imageView = m_lut3D[i] ? m_lut3D[i]->srgbView() : VK_NULL_HANDLE;
 	}
 
 	if (!m_target->isYcbcr())

